cmake/CPackSetup.cmake

```diff
@@ -0,0 +1,53 @@
+# package info
+set(CPACK_PACKAGE_NAME atsdk)
+set(CPACK_PACKAGE_DESCRIPTION "Atsign's atSDK library")
+set(CPACK_PACKAGE_VERSION 0.3.0)
+set(CPACK_PACKAGE_VENDOR_NAME atsign-foundation)
+
+# cmake configuration
+set(CPACK_CMAKE_GENERATOR "Unix Makefiles")
+set(
+  CPACK_PACKAGE_INSTALL_DIRECTORY
+  "${CMAKE_CURRENT_SOURCE_DIR}/package-source"
+)
+
+# static cpack configuration
+set(CPACK_PACKAGE_FILE_NAME atsdk-static)
+set(CPACK_GENERATOR ZIP TGZ)
+set(CPACK_INSTALL_CMAKE_PROJECTS ".;atsdk;ALL;/")
+
+# source cpack configuration
+set(CPACK_SOURCE_PACKAGE_FILE_NAME atsdk-source)
+set(CPACK_SOURCE_GENERATOR TGZ ZIP)
+
+set(CPACK_SOURCE_INSTALLED_DIRECTORIES "${CMAKE_CURRENT_SOURCE_DIR};/")
+
+# Yes, you need 4 * '\' to escape literal '.'
+set(
+  CPACK_SOURCE_IGNORE_FILES
```

Comment:
@cpswan please review the list of ignored files. I think I've chosen a decent middle ground of removing all project development specific things, but kept all things that might be relevant to a customer (e.g. examples)